mesheryctl/internal/cli/root/system/context.go

```diff
@@ -252,9 +263,14 @@ mesheryctl system context view --all
 * Usage of mesheryctl context view
 # ![context-view-usage](/assets/img/mesheryctl/context-view.png)
 	`,
-	Args:         cobra.MaximumNArgs(1),
+
 	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 && cmd.Flags().Lookup("all").Value.String() == "false" {
+			const errMsg = `Please provide a context name to view.
+Usage: mesheryctl system context view [context name]`
+			return errors.New(utils.SystemContextSubError(fmt.Sprintf("%s\n", errMsg), "context view"))
+		}
```

Comment:
Still won't work. This command can also be used with `--context` flag. Eg. `mesheryctl system context view --context context-name`